stats/src/test/java/io/airlift/stats/TestQuantileDigest.java

```diff
@@ -161,9 +161,9 @@ public void testQuantileLowerBound()
 
         assertThat(digest.getQuantileLowerBound(0.0)).isEqualTo(1);
         for (int i = 1; i <= 10; i++) {
-            assertThat(digest.getQuantileLowerBound(i / 10.0) <= i * 10).isTrue();
```

Comment:
Why? 10 * 10 fits in an int alright.

Reply:
yeah but getQuantileLowerBound returns a long

Reply:
```
[ERROR] /Users/mateusz.gajewski/Projects/src/github.com/airlift/airlift/stats/src/test/java/io/airlift/stats/TestQuantileDigest.java:[164,68] [NarrowCalculation] This product of integers could overflow before being implicitly cast to a long.
[ERROR]     (see https://errorprone.info/bugpattern/NarrowCalculation)
[ERROR]   Did you mean 'assertThat(digest.getQuantileLowerBound(i / 10.0) <= i * 10L).isTrue();'?
```